src/components/NcListItem/NcListItem.vue

```diff
@@ -470,6 +470,11 @@ export default {
 			default: '#',
 		},
 
+		target: {
+			type: String,
+			required: false,
```

Comment:
Any reason? The property is not required and when not provided would be undefined which would result in the correct evaluation of the target.

Is this a code style preference?

Reply:
To fix the eslint warning `Prop 'target' requires default value to be set`.
Because every non required props need a default value to be set.

`''` is considered falsy so it will work, but you could also set default to `null` if you like.

Reply:
> `''` is considered falsy so it will work, but you could also set default to null if you like.

Definitely.